test/cypress/integration/components/filter-blog-posts.js

```diff
@@ -0,0 +1,300 @@
+  } );
+  it( 'Select multiple topics', () => {
+    // When I select five checkboxes in the Topic list
+    filter.clickTopic( 'Student loans' );
+    filter.clickTopic( 'Financial Education' );
+    filter.clickTopic( 'Mortgages' );
+    filter.clickTopic( 'Consumer complaints' );
+    filter.clickTopic( 'Financial well-being' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results tagged with at least one of the two selected topics
+    page.notification().should( 'be.visible' );
+    page.resultsContent().should( 'contain', 'financial' );
+    // And the page url should contain "topics=student-loans"
+    cy.url().should( 'include', 'topics=student-loans' );
+    // And the page url should contain "topics=financial-education"
+    cy.url().should( 'include', 'topics=financial-education' );
+    // And the page url should contain "topics=mortgages"
+    cy.url().should( 'include', 'topics=mortgages' );
+    // And the page url should contain "topics=consumer-complaints"
+    cy.url().should( 'include', 'topics=consumer-complaints' );
+    // And the page url should contain "topics=financial-well-being"
+    cy.url().should( 'include', 'topics=financial-well-being' );
+  } );
+  it( 'Type-ahead topics', () => {
+    // When I type "mortgage" in the topic input box
+    filter.clickTopic( 'Mortgages' );
+    // Then the list of topics should show only tags that contain the word "mortgage"
+    page.resultsContent().should( 'contain', 'mortgage' );
+    // And when I select a topic in the list
+    filter.clickTopic( 'Servicemembers' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results tagged with the selected topic
+    page.notification().should( 'be.visible' );
+    page.resultsContent().should( 'contain', 'service' );
+    // And the page url should contain "topics=mortgages"
+    cy.url().should( 'include', 'topics=mortgages' );
+    // And the page url should contain "topics=servicemembers"
+    cy.url().should( 'include', 'topics=servicemembers' );
+  } );
+  it( 'Select category and topic', () => {
+    // When I select a checkbox in the Category list
+    filter.checkCategoryId( 'policy_compliance' );
+    // When I select a checkbox in the Topic list
+    filter.clickTopic( 'Students' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results that are both in the selected category and tagged with the selected topic
+    page.notification().should( 'be.visible' );
+    page.resultsContent().should( 'contain', 'student' );
+    page.resultsContent().should( 'contain', 'policy' );
+    // And the page url should contain "categories=policy_compliance"
+    cy.url().should( 'include', 'categories=policy_compliance' );
+    // And the page url should contain "topics=students"
+    cy.url().should( 'include', 'topics=students' );
+  } );
+  it( 'Clear and hide filters', () => {
```

Comment:
This test is still failing intermittently, and I think it has to do with some extraneous steps that I will suggest removing in individual comments below. If we want a test for hiding filters, let's not pair it up with clearing filters.

```suggestion
  it( 'Clear filters', () => {
```

Reply:
I agree as I see this failing as well so I created separate Clear and Hide filters tests.